dkg/dkg.go

```diff
@@ -246,6 +262,15 @@ func Run(ctx context.Context, conf Config) (err error) {
 	if err != nil {
 		return err
 	}
+
+	// Sign, exchange K1 signatures over Lock Hash
+	k1LockHashSigs, err := lhk1Bcast.exchange(ctx, lock.LockHash, key, nodeIdx)
```

Comment:
`lhk1Bcast` is such a hard thing to read.... the field in the lock hash is called `node_signatures`, what about sticking with that naming? `nodeSigs.exchange`?